src/test/java/racing/domain/CarTest.java

```diff
@@ -1,29 +1,47 @@
 package racing.domain;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
 
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.CsvSource;
+import racing.exception.RacingException;
 
 public class CarTest {
+  private final String NAME = "eun";
+
   private Car car;
+
   @BeforeEach
-  void init(){
-    car = new Car(0);
+  void init() {
+    car = new Car(NAME);
   }
 
   @Test
-  void carStartsAtZero(){
-    assertThat(car.getPosition()).isEqualTo(0);
+  void carStartsAtZero() {
+    assertThat(car.getPosition().getPosition()).isEqualTo(0);
   }
 
   @ParameterizedTest
-  @CsvSource({"4,true", "5,true", "0,false", "3,false"})
-  void carGoIfMoreThan4(int randomValue, boolean expected){
-    boolean result = car.move(randomValue);
-    assertThat(result).isEqualTo(expected);
-    assertThat(car.getPosition()).isEqualTo(randomValue >= 4 ? randomValue : 0);
+  @CsvSource({"4,1", "9,1", "0,0", "3,0"})
+  void carGoIfMoreThan4(int randomValue, int expected) {
+    car.move(randomValue);
+    assertThat(car.getPosition().getPosition()).isEqualTo(expected);
```

Comment:
```suggestion
    assertThat(car).isEqualTo(new Car(NAME, expected));
```
테스트 코드의 검증 또한 값을 꺼내기 보다 객체 자체로 비교하도록 의식적인 연습을 해보면 어떨까?

Reply:
말씀주신대로 객체끼리 비교하니 get을 사용해야 할 일이 줄어들었습니다.